lms/djangoapps/learner_recommendations/tests/test_utils.py

```diff
@@ -209,3 +211,18 @@ def test_recommend_only_active_courses(
             expected_recommendations.append(self._mock_get_course_data(course_key))
 
         assert filtered_courses == expected_recommendations
+
+
+@ddt.ddt
+class TestGetCrossProductRecommendationsMethod(TestCase):
+    """Test for get_cross_product_recommendations method"""
+
+    @ddt.data(
+        ("HKUx+FinTechT1x", ["ColumbiaX+BC24FNTC", "MITx+BLN"]),
+        ("RITx+CYBER501x", ["UniversityofUtah+BC24CYB", "HarvardX+CYB"]),
+        ('NoKeyAssociated', None)
+    )
```

Comment:
Can we update these course keys to some random test keys? We should avoid using actual course keys in the tests.